.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will run all checks required for a PR to be merged.
+
+name: Build and Test ci
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.0.0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "18.16.1"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Check linting and formatting
+        run: |
+          if yarn lint && yarn format --check; then
+            echo "Linting and formatting checks passed."
+          else
+            echo "Linting or formatting checks failed. Please fix the issues."
+            exit 1
+          fi
+
+      - name: Create and build generated code
```

Comment:
This won't succeed because the generator requires user input and has no way to accept the defaults.

Suggest for now you we don't run the generator and instead just build `recipe/next` because this is (manually, see #10) kept in sync with the generator.

Alternatively, we could remove this step for now.

Reply:
Ah I see, Sure have added the changes